vtr_flow/scripts/run_vtr_task.py

```diff
@@ -227,7 +227,7 @@ def vtr_command_main(arg_list, prog=None):
 
         config_files = [find_task_config_file(task_name) for task_name in task_names]
         configs = []
-        common_task_prefix = None  # common task prefix to shorten task names
+        common_task_prefix = ""  # common task prefix to shorten task names
```

Comment:
It looks like this changes the functionality of the script.  Shouldn't common_task_prefix be initialized to config.task_name instead of ""?  And with that change, I think the if / else checking if common_task_prefix is None can be deleted (just always do the else).

Reply:
I think the definition of `common_task_prefix` initializes by empty string needs to be outside of the for loop since the 'shorten_task_names' function uses this variable. By that, I mean the value of config.task_name is specified inside the loop so we can not initialize the `common_task_prefix` with the value of config.task_name outside of the loop. However, you are right about the if / else expression, the if expression should change to 'if not common_task_prefix' to follow the previous path. Sorry for this mistake

Reply:
Please correct me if I am wrong

Reply:
Ah, you're right -- didn't read that carefully enough.  Yes, I think the fix will work.